src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						object.view = object.view === null ?
```

Comment:
LOL Obviously is, because I just noticed that the code is actually wrong. Should check whether `data.view !== null`, no? One sec...